apps/payments/next/config/index.ts

```diff
@@ -1,11 +1,17 @@
 import 'reflect-metadata';
+import 'server-only';
 import { Type } from 'class-transformer';
-import { IsString, ValidateNested, IsDefined } from 'class-validator';
+import { IsString, ValidateNested, IsDefined, IsUrl } from 'class-validator';
 import {
   RootConfig as NestAppRootConfig,
   validate,
 } from '@fxa/payments/ui/server';
 
+class CspConfig {
+  @IsUrl()
+  accountsStaticCdn!: string;
+}
+
 class AuthJSConfig {
   @IsString()
```

Comment:
Unrelated, but should this and L19-20 use this new `@IsUrl()` decorator too?

```diff
-  @IsString()
+  @IsUrl()
  issuerUrl!: string;

-  @IsString()
+  @IsUrl()
  wellKnownUrl!: string;
```

Reply:
^ Good question. Curious about this too.

Reply:
Yup makes sense.

Reply:
This is very hard to grep, apparently, but this might be an assortment of unreliable results where we're using `@IsString()` that could be converted to `@IsUrl()`:
```sh
git grep -A3 "@IsString()" | grep -E "( string)" | grep -Ei "(Url|Uri|redirectTo)"

apps/payments/next/config/index.ts-  issuerUrl!: string;
apps/payments/next/config/index.ts-  wellKnownUrl!: string;
packages/fxa-settings/src/models/integrations/client-info.ts-  imageUri: string | undefined;
packages/fxa-settings/src/models/integrations/client-info.ts-  redirectUri: string | undefined;
packages/fxa-settings/src/models/integrations/oauth-integration.ts-  imageUri: string | undefined;
packages/fxa-settings/src/models/integrations/oauth-integration.ts-  declare redirectTo: string | undefined;
packages/fxa-settings/src/models/integrations/oauth-integration.ts-  redirectUrl: string | undefined;
packages/fxa-settings/src/models/integrations/oauth-integration.ts-  redirectUri: string | undefined;
packages/fxa-settings/src/models/integrations/signin-signup-info.ts-  redirectUri: string | undefined;
packages/fxa-settings/src/models/integrations/signin-signup-info.ts-  redirectTo: string | undefined;
packages/fxa-settings/src/models/integrations/web-integration.ts-  redirectTo: string | undefined;
packages/fxa-settings/src/models/pages/signin/query-params.ts-  redirectTo: string | undefined = undefined;
```

Although guessing everything in fxa-settings is out of scope here.